src/server/controllers/collectives.js

```diff
@@ -28,3 +29,107 @@ export async function info(req, res, next) {
 
   res.send(response);
 }
+
+const query = `
+query($slug: String){
+  account(slug: $slug) {
+    memberOf(role: HOST){
```

Comment:
Make sure to filter COLLECTIVE, otherwise you are ending up with empty entries later when doing ` ... on Collective {`

``` memberOf(role: HOST, accountType: COLLECTIVE) {```

Reply:
Based on the update to the `API`. We can make this change.
```
host(slug: $slug) {
  memberOf(role: HOST, accountType: COLLECTIVE, limit: $limit) {}
}
```